.gitignore

```diff
@@ -13,6 +13,7 @@ devenv
 .DS_Store
 localbin
 nohup.out
+docker/

```

Comment:
If this ignore entry applies to your local development only, best to add it to your user profile .gitignore